admin-ui/src/pages/login/LoginForm.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react'
+import logo from '../../assets/logo-black.svg';
+import { Form, Icon, Input, Button } from 'antd';
+import { createFormField } from 'rc-form';
+
+class LoginForm extends Component {
+  render() {
+
+    const { getFieldDecorator } = this.props.form;
+    return (
+      <div className="card">
+        <div className="content">
+          <img className="logo" src={logo} /><br />
+          <p className="about">If you've got an idea, let's space it up !</p>
+
+          <Form onSubmit={this.props.handleSubmit} className="login-form">
```

Comment:
Dont submit directly. Prevent the default form submit action and then call the props.handleSubmit